src/guide/code-owners.md

```diff
@@ -126,40 +163,37 @@ This list is intended to make it easier to identify which email list to include
 * `java.xml` – Core Libs
 * `java.xml.crypto` – Security
 * `jdk.accessibility` – Client Libs
-* `jdk.aot` – HotSpot Compiler
 * `jdk.attach` – Serviceability
 * `jdk.charsets` – I18n, Core Libs
-* `jdk.compiler` – LangTools
+* `jdk.compiler` – Javac
 * `jdk.crypto.cryptoki` – Security
 * `jdk.crypto.ec` – Security
 * `jdk.crypto.mscapi` – Security
-* `jdk.crypto.ucrypto` – Security
-* `jdk.dynalink` – LangTools
-* `jdk.editpad` – LangTools
+* `jdk.dynalink` – JDK Tools
+* `jdk.editpad` – JShell
 * `jdk.hotspot.agent` – Serviceability
 * `jdk.httpserver` – Net
-* `jdk.incubator.foreign` – LangTools
-* `jdk.incubator.httpclient` – Net
-* `jdk.incubator.vector` – HotSpot Compiler
-* `jdk.internal.ed` – LangTools
+* `jdk.incubator.concurrent` – Core Libs
+* `jdk.incubator.vector` – Compiler
+* `jdk.internal.ed` – JShell
 * `jdk.internal.jvmstat` – Serviceability
-* `jdk.internal.le` – LangTools
-* `jdk.internal.opt` – LangTools
-* `jdk.internal.vm.ci` – HotSpot Compiler
-* `jdk.internal.vm.compiler` – HotSpot Compiler
-* `jdk.internal.vm.compiler.management` – HotSpot Compiler
-* `jdk.jartool` – LangTools
-* `jdk.javadoc` – LangTools
+* `jdk.internal.le` – JShell
+* `jdk.internal.opt` – JDK Tools
+* `jdk.internal.vm.ci` – Compiler
+* `jdk.internal.vm.compiler` – Compiler
+* `jdk.internal.vm.compiler.management` – Compiler
+* `jdk.jartool` – JDK Tools
```

Comment:
and Security because this includes `jarsigner`.

Reply:
Fixed.